README.md

```diff
@@ -251,6 +252,11 @@ format:
 Returns a stream which has all the clientIds subscribed to the
 specified topic
 
+<a name="destroy"></a>
+### instance.destroy(cb(err))
+
+Destroy current persistence. Returns `err` if any
```

Comment:
I don’t think this returns err. It calls the callback with err

Reply:
You right sorry I explained it wrongly